test/python/test_public_keys_api.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import absolute_import
+
+import unittest
+
+import openapi_client
+
+from . import api_config
+
+
+class TestPublicKeysApi(api_config.ConfiguredTest):
+    """PublicKeysApi unit test stubs"""
+
+    def setUp(self):
+        self.api = openapi_client.api.public_keys_api.PublicKeysApi(self.client)
+
+    def test_show_public_keys(self):
+        """Test case for show_public_keys
+
+        Shows all public keys for a resource.
+        """
+        resp = self.api.show_public_keys(self.account, 'Variable', 'one/password')
+
+        self.assertIsInstance(resp, str)
```

Comment:
Not necessarily breaking, but wondering if this assertion actually confirms that the API call worked. Seems a bit too general compared to a lot of the other test cases.

Reply:
This test is definitely lighter then others. However I check that the return type is a string because this endpoint is supposed to return a plain text body so I am just making sure that the client returns the proper type. These are just integration tests so I didn't want to add any checks that were too complicated in.